applications/band-grants.md

```diff
@@ -0,0 +1,164 @@
+
+
+## Development Status :open_book:
+
+- links to your research diary, blog posts, articles, forum discussions or open GitHub issues,
+- references to conversations you might have had related to building this template,
+    - Band Partners: https://www.bandpartners.io/
+    - Blog: https://blog.bandprotocol.com/
+    - Github: https://github.com/bandprotocol/
+
+## Development Roadmap :nut_and_bolt:
+
+This section should break the development roadmap down into milestones and deliverables. Since these will be part of the agreement, it helps to describe _the functionality we should expect in as much detail as possible_, plus how we can verify and test that functionality. Whenever milestones are delivered, we refer to this document to ensure that everything has been delivered as expected.
+
+Below we provide an **example roadmap**.
+
+> :exclamation: If any of your deliverables is based on somebody else's work, make sure you work and publish _under the terms of the license_ of the respective project and that you **highlight this fact in your milestone documentation** and in the source code if applicable! **Teams that submit others' work without attributing it will be immediately terminated.**
+
+### Overview
+
+- **Total Estimated Duration:** 3-4 months
+- **Full-Time Equivalent (FTE):**  2
+- **Total Costs:** 80,000 USD
+
+### Milestone 
+
+*NB: do we need to add infra fees?*
+
+- **Estimated duration:** 3-4 months
+- **FTE:**  2
+- **Costs:** 80,000 USD
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one. 
+
+|  Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| **0b.** | Documentation | We will provide inline documentation of the code, a basic tutorial on how to use the smart contract and an example consumer contract that shows a practical example on how to use the smart contract|
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests.|
+| **0d.** | Docker | N/A for smart contracts|
+| 0e. | Article | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). (Content, language and medium should reflect your target audience described above.) |
+| 1. | Data Source | Data sources on BandChain which will be the core of getting data that will be delivered to Astar |
+| 2. | Oracle Script | An oracle script on BandChain which will verify and aggregate data from the data sources on BandChain|
+| 3. | BandChain Requester | A service that will prepare and request data from BandChain to be sent to Astar | 
+| 4. | Astar ink! Relayer | A service that will prepare and execute data to be bridged over to Astar|
+| 5. | ink! Signer | A seperate but crucial service that will used concurrently to Astar to send over data |
```

Comment:
What's the relationship between the relayer and the signer? Could you elaborate on the signing process, i.e. how is the security model? Does it use threshold signature to create efficient transactions?